alg/crc32c.c

```diff
@@ -63,7 +63,7 @@ init(void)
 
 	/* Fill in tables. */
 	for (i = 0; i < 256; i++) {
-		r = reverse(i);
+		r = reverse(i & 0xffff);
```

Comment:
but what if "what we're trying to do" doesn't make sense?

... ok, I'll go order pizza now.